src/theme/styles/fonts.ts

```diff
@@ -24,5 +26,45 @@ const fontFace = css`
     font-style: normal;
   }
 `
+export interface Fonts {
+  fontDefault: string
+  fontArial: string
+  fontSizeNormal: string
+  fontSizeSmall: string
+  fontSizeLarge: string
+  fontSizeXLarge: string
+  fontSizeXXLarge: string
+  fontWeightNormal: string
+  fontWeightMedium: string
+  fontWeightBold: string
+}
+
+const DEFAULT_FONT = {
+  fontDefault: '"Inter", "Helvetica Neue"',
+  fontArial: 'Arial',
+  fontSizeNormal: '1.2rem',
+  fontSizeSmall: '1.1rem',
+  fontSizeLarge: '1.3',
+  fontSizeXLarge: '1.4rem',
+  fontSizeXXLarge: '1.5rem',
+  fontWeightNormal: '400',
+  fontWeightMedium: '500',
+  fontWeightBold: '700',
+}
+
+export function getFonts(mode: Theme): Fonts {
+  let FONT: Fonts
+
+  switch (mode) {
+    case Theme.LIGHT:
+    case Theme.DARK:
+      FONT = DEFAULT_FONT
+      break
```

Comment:
Redundant, you might as well use only `default`, or skip the switch altogether and always return `DEFAULT_FONT`

Reply:
true, true, will fixito